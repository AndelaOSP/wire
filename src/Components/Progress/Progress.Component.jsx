import React from 'react';
import CircularProgress from 'material-ui/CircularProgress';

// styles
import './Progress.scss';

const CircularProgressIndicator= () => (
  <div className="progress-container">
    <CircularProgress size={50} thickness={5} color="#127dc5" />
  </div>
);

export default CircularProgressIndicator;
