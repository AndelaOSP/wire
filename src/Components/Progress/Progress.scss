@import '../../theme.scss';

.progress-container {
    display: flex;
    flex-direction: row;
    justify-content: center;
    margin: auto;
    padding: 1rem;
}
