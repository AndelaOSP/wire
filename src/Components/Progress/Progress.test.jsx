import React from 'react';
import { shallow } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import Progress from './Progress.Component';

describe('Progress component', () => {
  it('should render without crashing', () => {
    const progress = shallow(<Progress />);
    expect(shallowToJSON(progress)).toMatchSnapshot();
  });

  it('should render CircularProgress component', () => {
    const progress = shallow(<Progress />);
    const progressContainer = progress.find('.progress-container');
    const circularProgress = progress.find('CircularProgress');
    expect(progressContainer.length).toEqual(1);
    expect(circularProgress.length).toEqual(1);
  });
});
