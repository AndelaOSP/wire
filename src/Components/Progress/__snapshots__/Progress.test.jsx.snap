// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress component should render without crashing 1`] = `
<div
  className="progress-container"
>
  <CircularProgress
    color="#127dc5"
    max={100}
    min={0}
    mode="indeterminate"
    size={50}
    thickness={5}
    value={0}
  />
</div>
`;
